config/secrets.yml

```
@@ -43,6 +43,8 @@ default: &default
       throttle:
         max_requests: <%= ENV["THROTTLING_MAX_REQUESTS"]&.to_i || 100 %>
         period: <%= ENV["THROTTLING_PERIOD"]&.to_i || 60 %>
+    participatory_processes:
+      sort_by_date: <%= ENV["SORT_PROCESSES_BY_DATE"] || false %>
```

Env variable have type String, you can cast it to TrueClass or FalseClass by comparing value with `"true"`

```suggestion
      sort_by_date: <%= ENV.fetch("SORT_PROCESSES_BY_DATE", "false") == "true" %>
```

---

ok, thx for your suggestion @Quentinchampenois, I am going to apply it !